draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -528,6 +528,11 @@ and expandedKey format are technically redundant when using the seed-only
 format, they MAY be included to enable keypair consistency checks during
 import operations.
 
+When parsing the private key, the ASN.1 tag explicitly indicates which
+variant of CHOICE is present. Implementations should use tag value 0x80
+for seed only, 0x04 for key only, and 0x30 for both to parse the private
+key, rather than any other heuristic like length of the OCTET STRING.
```

Comment:
```suggestion
variant of `CHOICE` is present. Implementations should use tag `UNIVERSAL IMPLICIT [0]`
(raw value `0x80`) for `seed`, `OCTET STRING` (`0x04`) for `expandedKey`, and
`SEQUENCE` (`0x30`) for `both` to parse the private key, rather than any other heuristic like length of the `OCTET STRING`.
```